warehouse/templates/search/results.html

```diff
@@ -128,81 +128,94 @@ <h2 class="no-top-padding">Filter Projects</h2>
     </div>
 
     <div class="left-layout__main">
+      {% if not term and not applied_filters %}
+      <div class="callout-block no-top-margin">
+        <p>Enter a search query above, or select a filter from the list of classifiers on the left.</p>
+        <p>You can combine searches and classifier filters. Examples:</p>
+        <ul class="no-bottom-margin">
+          <li><a href="/search/?q=&o=&c=Programming+Language+%3A%3A+Python+%3A%3A+3">New Python 3 project releases</a></li>
```

Comment:
I wasn't sure the best way to make these examples - so I've just hardcoded them for now :/
could someone please advise?

Reply:
Figured it was easier to "show" than "tell" so I fixed this in 16423df, take a look. 🙂